src/components/LandingPage/Promo/GitStarPromoSection/GitStarPromoSection.tsx

```diff
@@ -8,14 +9,15 @@ import { useStarsHover } from './useStarsHover'
 export const GitStarPromoSection = () => {
 	const { starCount, hasStars } = useGitStars()
 	const { handleHoverOut, handleHoverOver, isHoveredOver } = useStarsHover()
+	const prefersReducedMotion = usePrefersReducedMotion()
 
 	return (
 		<div className={classes.root}>
 			<div className={classes.textContainer}>
 				<h2 className={classes.sectionTitle}>Give us a ⭐ on GitHub</h2>
 				<a
-					onMouseOver={handleHoverOver}
-					onMouseOut={handleHoverOut}
+					onMouseOver={prefersReducedMotion ? undefined : handleHoverOver}
```

Comment:
change the ternaries to:
{!prefersReducedMotion ? handleHoverOver : undefined}

Reply:
Could you provide more information on why writing this ternary expression in the mentioned way would be better? Do you think it will be more readable or are there other reasons?

Reply:
It will be more readable.
Not a big deal, but that's my preferance.